infrastructure/ansible/roles/ats/defaults/main.yml

```diff
@@ -23,15 +32,25 @@ ort_traffic_ops_url:
 ort_traffic_ops_username:
 ort_traffic_ops_password:
 
-# ORT cronjobs to deploy
+# t3c cronjobs to deploy
 ort_crontab:
   # Perform configuration polling that doesn't require a restart on a 20 minute +- dispersion interval
   syncds:
     schedule: '0,20,40 * * * *'
     user: root
-    job:  "traffic_ops_ort.pl syncds warn {{ ort_traffic_ops_url }} '{{ ort_traffic_ops_username }}:{{ ort_traffic_ops_password }}' --login_dispersion=35 --dispersion=420 &>/tmp/ort/syncds.log"
+    job: >
+      {% if ort_version is not defined or ort_version == omit or ort_version >= 6.0.0.11351 %}
+        t3c apply --run-mode=syncds --log-location-warning=stderr --log-location-error=stderr --traffic-ops-url='{{ ort_traffic_ops_url }}' --traffic-ops-user='{{ ort_traffic_ops_username }}' --traffic-ops-password='{{ ort_traffic_ops_password }}' --login-dispersion=35 --dispersion=420 &> /tmp/trafficcontrol-cache-config/syncds.log
+      {% else %}
+        traffic_ops_ort.pl syncds warn {{ ort_traffic_ops_url }} '{{ ort_traffic_ops_username }}:{{ ort_traffic_ops_password }}' --login_dispersion=35 --dispersion=420 &>/tmp/ort/syncds.log
+      {% endif %}
   # Perform a check for user requested content invalidations every minute
   reval:
     schedule: '1-19,21-39,41-59 * * * *'
     user: root
-    job:  "traffic_ops_ort.pl revalidate warn {{ ort_traffic_ops_url }} '{{ ort_traffic_ops_username }}:{{ ort_traffic_ops_password }}' --login_dispersion=35 &>/tmp/ort/reval.log"
+    job: >
```

Comment:
See above whitespace and version quoting

Reply:
Fixed